### ISSUE TITLE
ti:commentary

### ISSUE THREAD

**Issue by sonofmun** [Capitains/Capitains.github.io#17]
Consider introducing a ti:commentary type to be able to keep the front and back matter of an edition in relation to that edition.

**Comment by PonteIneptique** [comment]
New system to be born : Same nodes than edition or translation, + new relationship node

```xml
<ti:commentary urn="urn:cts:greekLit:phi001.phi002.1st1kGrc-c-ger1" xml:lang="ger">
<!-- Adding the C is just a recommandation -->
<ti:title xml:lang="eng">Introduction to Edition blabla</ti:title>
<ti:description xml:lang="ger">German description</ti:description>
<ti:about urn="urn:cts:greekLit:phi001.phi002.1st1kGrc-ger1"/>
</ti:commentary>
```

If not specific to an edition : 

```xml
<ti:commentary urn="urn:cts:greekLit:phi001.phi002.1st1kGrc-c-ger1" xml:lang="ger">
<!-- Adding the C is just a recommandation -->
<ti:title xml:lang="eng">Introduction to Edition blabla</ti:title>
<ti:description xml:lang="ger">German description</ti:description>
<ti:about urn="urn:cts:greekLit:phi001.phi002"/>
</ti:commentary>
```

**Comment by PonteIneptique** [comment]
Thoughts ?

**Comment by PonteIneptique** [comment]
In DTS Scheme, Edition Specific Commentary would be children of Edition, while Text Wide Commentary would be children

**Comment by PonteIneptique** [comment]
Introducing a type would be great but it means having an ontology :')

**Comment by sonofmun** [comment]
That looks good. Do we also want Dublin Core metadata for these? And, if so, which fields?

**Comment by PonteIneptique** [comment]
cpt:metadata would be the same.
cpt:metadata might not come into CapiTainS straight away though.

**Comment by PonteIneptique** [comment]
Matt, could you edit https://github.com/Capitains/Capitains.github.io/blob/master/pages/guidelines.md to reflect this addition ?

**Comment by sonofmun** [comment]
Done.

**Comment by PonteIneptique** [comment]
Fixed by #19
